### PR TITLE
Add parameter description for 2012 version

### PR DESCRIPTION
**Pull request by velkovb**
Add parameter description for 2012 version and adjust formatting for all version so it is uniform.

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by tseward** [comment]
Changes look good.

File: docset/winserver2012-ps/activedirectory/Add-ADDomainControllerPasswordReplicationPolicy.md [hunk 1]
```diff
@@ -162,12 +208,32 @@ Aliases:
...
 Position: 0
 Default value: None
-Accept pipeline input: False
+Accept pipeline input: True
 Accept wildcard characters: False
 ```
 
 ### -Server
-{{Fill Server Description}}
+Specifies the Active Directory Domain Services instance to connect to, by providing one of the following values for a corresponding domain name or directory server.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Add-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
Specifies the Active Directory Domain Services instance to connect to by providing one of the following values for a corresponding domain name or directory server.
```

File: docset/winserver2012-ps/activedirectory/Add-ADDomainControllerPasswordReplicationPolicy.md [hunk 2]
```diff
@@ -162,12 +208,32 @@ Aliases:
...
+The service may be any of the following:  Active Directory Lightweight Domain Services, Active Directory Domain Services or Active Directory snapshot instance.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Add-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
The service may be any of the following:  Active Directory Lightweight Domain Services, Active Directory Domain Services, or Active Directory snapshot instance.
```

File: docset/winserver2012-ps/activedirectory/Add-ADDomainControllerPasswordReplicationPolicy.md [hunk 3]
```diff
@@ -198,23 +264,21 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Add-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [About CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2012r2-ps/addsadministration/Add-ADDomainControllerPasswordReplicationPolicy.md [hunk 4]
```diff
@@ -272,12 +272,12 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/addsadministration/Add-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [About CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```

**Comment by e0i** [comment]
@get-itips Any additional comments?

**Comment by traya1** [comment]
Hi @e0i - We understand that these [PR criteria](https://review.docs.microsoft.com/en-us/office-authoring-guide/pr-criteria?branch=master) are new, and we appreciate your patience with the process. 

**Following are blocking issues that need to be resolved before we can merge:**

docset/windows/addsadministration/add-addomaincontrollerpasswordreplicationpolicy.md
- merge conflict

***

**Reference**

[Criteria used for pull request reviews](https://review.docs.microsoft.com/en-us/office-authoring-guide/pr-criteria?branch=master)
- The PR is free of merge conflicts. If a merge conflict exists, then see [Resolve merge conflicts in Git and GitHub](https://review.docs.microsoft.com/en-us/help/contribute/resolve-merge-conflicts?branch=master).

***

Thanks

**Comment by e0i** [comment]
Merge conflict(s) have been resolved.

**Pull request merged**